crates/rnote-ui/src/appwindow/mod.rs

```diff
@@ -104,6 +104,30 @@ impl RnAppWindow {
         self.imp().overlays.get()
     }
 
```

Comment:
Let's move these function below `init()` for consistency. I always followed the convention getters/setters -> `init()` -> other

Reply:
fixed